### PR TITLE
fix: installed nodejs 18, configured redis, removed duplicate monit onfig

### PR DESCRIPTION
**Pull request by apinter**
Test:

```bash
$ podman build --format=docker . -t xen-testing
$ podman run -itd --name xentest --cap-add=SYS_ADMIN -e HTTP_PORT=80 -v xo_data:/var/lib/xo-server:z -v redis-data:/var/lib/redis:z  -p 8881:80 localhost/xen-testing:latest
```

### CODE REVIEW

**Review by ctlinux — APPROVED** [review]
Thank you for your contribution!

File: Dockerfile [hunk 1]
```diff
@@ -47,13 +51,21 @@ RUN ln -sf /proc/1/fd/1 /var/log/valkey/valkey.log && \
 # Healthcheck
 ADD healthcheck.sh /healthcheck.sh
 RUN chmod +x /healthcheck.sh
+
+RUN mkdir /etc/redis && \
+    mkdir var/log/redis && \
+    touch -a /var/log/redis/redis-server.log
```
Comment (Conan-Kudo) [inline Dockerfile:57]: This should be a symlink to the valkey log file.

File: Dockerfile [hunk 2]
```diff
@@ -47,13 +51,21 @@ RUN ln -sf /proc/1/fd/1 /var/log/valkey/valkey.log && \
...
+ADD conf/redis.conf /etc/redis/redis.conf
```
Comment (Conan-Kudo) [inline Dockerfile:58]: This file would not be used.

File: Dockerfile (hunk 1 above)
Comment (Conan-Kudo) [reply Dockerfile:57]: `ln -sf /proc/1/fd/1 /var/log/redis/redis-server.log`

**Pull request merged**